src/models/channel-parameter.ts

```diff
@@ -1,3 +1,11 @@
 import type { BaseModel } from "./base";
+import type { DescriptionMixinInterface, ExtensionsMixinInterface } from "./mixins";
+import type { SchemaInterface } from "./schema";
 
-export interface ChannelParameterInterface extends BaseModel {}
+export interface ChannelParameterInterface extends BaseModel, DescriptionMixinInterface, ExtensionsMixinInterface {
+  id(): string;
+  hasSchema(): boolean;
```

Comment:
If this should stay, should be added to https://github.com/asyncapi/parser-api/blob/master/docs/v1.md#channelparameter